### DISCUSSION TITLE
even though I have  setup cachix but still have to building ```wine-ge-full-Proton8-26```, is this expected?

### DISCUSSION THREAD

**Discussion by deng232** [fufexan/nix-gaming#171] [Q&A]
even though I have  setup cachix but still have to building ```wine-ge-full-Proton8-26```, is this expected?

setup:
```
{ pkgs, config, inputs, ... }: {
  home.packages = with pkgs;[
    gamemode
    gamescope
    winetricks
    inputs.nix-gaming.packages.${pkgs.system}.wine-ge
  ];
  nix.settings = {
    substituters = ["https://nix-gaming.cachix.org"];
    trusted-public-keys = ["nix-gaming.cachix.org-1:nbjlureqMbRAxR1gJ/f3hxemL9svXaZF/Ees8vCUUs4="];
  };
}
```
it takes long time to build and virtualbox network rate is zero while it was building

**Comment by fufexan** [comment]
You seem to have the nix settings set by home manager. If you are on NixOS, you should set the substituter settings in the NixOS config.

If you only use home manager on a different distro, make sure you rebuild once without the package in the list, so that Nix has a chance to update its conf file first. Might be worth checking `~/.config/nix/nix.conf`.
